cumulus/steps/dev_tools/pipeline.py

```diff
@@ -213,3 +200,109 @@ def handle(self, chain_context):
         chain_context.template.add_resource(pipeline_service_role)
         chain_context.template.add_resource(generic_pipeline)
         chain_context.template.add_output(pipeline_output)
+
+    def get_default_bucket_policy_statements(self, pipeline_bucket):
+        bucket_policy_statements = [
+            awacs.aws.Statement(
+                Effect=awacs.aws.Allow,
+                Action=[
+                    awacs.s3.ListBucket,
+                    awacs.s3.GetBucketVersioning,
+                ],
+                Resource=[
+                    troposphere.Join('', [
+                        awacs.s3.ARN(),
+                        Ref(pipeline_bucket),
+                    ]),
+                ],
+            ),
+            awacs.aws.Statement(
+                Effect=awacs.aws.Allow,
+                Action=[
+                    awacs.s3.HeadBucket,
+                ],
+                Resource=[
+                    '*'
+                ]
+            ),
+            awacs.aws.Statement(
+                Effect=awacs.aws.Allow,
+                Action=[
+                    awacs.s3.GetObject,
+                    awacs.s3.GetObjectVersion,
+                    awacs.s3.PutObject,
+                    awacs.s3.ListObjects,
+                    awacs.s3.ListBucketMultipartUploads,
+                    awacs.s3.AbortMultipartUpload,
+                    awacs.s3.ListMultipartUploadParts,
+                    awacs.aws.Action("s3", "Get*"),
+                ],
+                Resource=[
+                    troposphere.Join('', [
+                        awacs.s3.ARN(),
+                        Ref(pipeline_bucket),
+                        '/*'
+                    ]),
+                ],
+            )
+        ]
+
+        return bucket_policy_statements
+
+    def get_bucket_policy(self, pipeline_bucket, bucket_policy_statements):
+        # name = "CFDeployerAccessToolsAccountBucket%s" % self.context.namespace
+        # raise ValueError(pipeline_bucket.title.__str__)
+        policy = troposphere.s3.BucketPolicy(
+            "PipelineBucketPolicy",
+            # "%sPolicy" % pipeline_bucket.__str__(),
+            Bucket=troposphere.Ref(pipeline_bucket),
+            PolicyDocument=awacs.aws.Policy(
+                Statement=bucket_policy_statements,
+
+                # [
```

Comment:
Remove commented out stuff?

Reply:
Yup. Probably should do this in a few other spots too.  I'll get 'er done when I'm back